src/modules/proto_bfd/proto_bfd.c

```diff
@@ -387,7 +387,10 @@ static const char *bfd_state[] = {
 static void bfd_request(bfd_state_t *session, request_t *request, fr_radius_packet_t *packet)
 {
 	memset(request, 0, sizeof(*request));
+	fr_pair_list_init(&request->control);
+	fr_pair_list_init(&request->state);
 	memset(packet, 0, sizeof(*packet));
+	fr_pair_list_init(&packet->vps);
```

Comment:
And this should really be initialised by the packet allocation or init functions

Reply:
Didn't see the `packet->vps` changes in here, but maybe I missed them?